src/main/java/core/basesyntax/AppMainPlane.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class AppMainPlane {
+    public static void main(String[] args) {
+        Plane.PlaneBuilder redPlaneBuilder = new Plane.PlaneBuilder("shelby")
+                .setColor("red")
+                .setWeight(1474);
+        Plane rxShelby = redPlaneBuilder.setModel("GT-350")
+                .setSpeed(231).setColor("red")
+                .build();
```

Comment:
this is more readable in case of a method pipeline
same for subsequent lines
```suggestion
        Plane rxShelby = redPlaneBuilder
                .setModel("GT-350")
                .setSpeed(231)
                .setColor("red")
                .build();
```